### PR TITLE
adding Nepal Disaster file to coordination section

### PR DESCRIPTION
**Pull request by Nick-Tallguy**
Althio - please have a read through. I'm investigating how we could add a link from the front page of the site to this file. Feel free to made amendments - you're grammar is better than mine.

### CODE REVIEW

File: _posts/en/coordination/0500-10-27-nepal.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Like you, we are keen to help as quickly as possible, and there is a real sense of urgency in what we do. But, the reason the Aid Organisations depend on us to provide the mapping data is because we take care to make sure that what we produce is of a high standard. We need you to take care to. Although it will delay you starting to update the map we need you to understand what is required of you. Do not rush in and make a mess of it, because it takes us longer to clear up bad work than it does to create it, and it may cause a care worker to have doubts about the dependability of what we produce. 
+
+We are constantly updating out software & training guides, but it does take time. The software in the Tasking Manager will be updated soon so that only qualified people can 'validate', but at the moment we need you to understand that validating should be left for the qualified few - please keep helping and you may become one of our validators in the future. **If you select a square and the screen mentions 'Review the work' then please release the square, because that is something that needs a 'validator'.**
```
Comment (althio) [inline _posts/en/coordination/0500-10-27-nepal.md]: - updating out software
+ updating our software

File: _posts/en/coordination/0500-10-27-nepal.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+We don't need you to read the whole of this site before you start, but we do need you to understand the basics:  
+
+1.  You need to read the Task Manager Instructions for any project you are trying to help for. They can be different, so make sure you read & understand them. There is a full description of how to use the [Tasking Manager here within LearnOSM](/en/coordination/tasking-manager/), and it includes how to get live help from an experienced mapper.  
+
+2.  Make sure you have zoomed in enough so that you can actually see what you should be tracing - [it and many other things are explained here.](/en/coordination/remote/#initial-view--josm)
+
+3.  Buildings have to be traced, and the shape of the tracing should match the shape of the building. [This section of LearnOSM](/en/coordination/remote-tracing/) explains this, and how to create roads and a road network.
+
+Thank you for reading this, and I hope you'll take the short time needed to make sure you do the important things right. We need all the help we can get, and hope that you can help us with this.
```
Comment (althio) [inline _posts/en/coordination/0500-10-27-nepal.md]: l.27 - and I hope
l.27 - and we hope

File: _posts/en/coordination/0500-10-27-nepal.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+category: coordination
+---
+
+#Nepal Disaster  
+
+![HotGuideLogo](http://hot.openstreetmap.org/sites/default/themes/hot/logo.png)
+
+Thank you for volunteering to help the Aid Organisations by providing mapping data for them. 
+
+Like you, we are keen to help as quickly as possible, and there is a real sense of urgency in what we do. But, the reason the Aid Organisations depend on us to provide the mapping data is because we take care to make sure that what we produce is of a high standard. We need you to take care to. Although it will delay you starting to update the map we need you to understand what is required of you. Do not rush in and make a mess of it, because it takes us longer to clear up bad work than it does to create it, and it may cause a care worker to have doubts about the dependability of what we produce.
```
Comment (althio) [inline _posts/en/coordination/0500-10-27-nepal.md]: l.15 "what we produce is of a high standard"
proposing:
l.15 "what we produce is useful and meets their needs."

**Comment by althio** [comment]
I am happy with that, see my minor proposals for amendments.
As we publish, I propose we ask feedback/review from other working groups (Activation / Communication).

**Comment by Nick-Tallguy** [comment]
Altho,

Thanks for that.  I agree.

If we can it would be good to get a link on the front page but I can't do
it from my phone and I'm on that until at least late this evening.

Nick
(OSM=Tallguy)

dodgy didgits as using a phone with a spell chequer.

Humanitarian OpenStreetMap Team Member

**Comment by althio** [comment]
Will you modify by new commits directly in this pull request?

> If we can it would be good to get a link on the front page

I propose (quick and dirty...) to modify `_posts/en/0100-01-01-index.md` from:
```
interspan: "More guides (currently in English)"
interspanlink: "https://github.com/hotosm/learnosm/wiki/English-Learning-Guides"
```
to:
```
interspan: "Nepal Disaster"
interspanlink: "http://learnosm.org/en/coordination/nepal-disaster/"
```

**Comment by Nick-Tallguy** [comment]
Althio,

Yes,  that would be brilliant.

Nick
(OSM=Tallguy)

dodgy didgits as using a phone with a spell chequer.

Humanitarian OpenStreetMap Team Member

**Comment by Nick-Tallguy** [comment]
Althio,

Sorry need to clarify.  Can you do it,  if not it may take a day or so
before I get the chance.

Nick
(OSM=Tallguy)

dodgy didgits as using a phone with a spell chequer.

Humanitarian OpenStreetMap Team Member

**Comment by Nick-Tallguy** [comment]
The interspanlink may need the mention of github.com

I'm not sure,  perhaps try it and modify if needed

Nick
(OSM=Tallguy)

dodgy didgits as using a phone with a spell chequer.

Humanitarian OpenStreetMap Team Member

**Comment by althio** [comment]
> Sorry need to clarify.  Can you do it,  if not it may take a day or so before I get the chance

Amendments to your text: Can I leave to you? Possibly with new commits on this current pull request.

Modification of the front page: I am a bit reluctant because I don't have access to a local repository for testing purpose right now.

Sorry for delaying this.

File: _posts/en/coordination/0500-10-27-nepal.md (hunk 1 above)
Comment (RAytoun) [inline _posts/en/coordination/0500-10-27-nepal.md]: We need you to take care too.

**Comment by Nick-Tallguy** [comment]
Just merging Japanese change - think this should work ok now as have tried it out locally. I'd missed one of the files the first time. This will also place the Nepal disaster page in the coordination section

**Comment by Nick-Tallguy** [comment]
@althio the 3 files that I changed to implement the Japanese change of folder were (on my computer, but you'll understand)  

~/HOT/MyLearnOSM/all-translation.json
~/HOT/MyLearnOSM/_config.yml
~/HOT/MyLearnOSM/_layouts/default.html

**Comment by althio** [comment]
:+1:

**Pull request merged**